hdfs/glob.py

```diff
@@ -0,0 +1,120 @@
+import fnmatch
```

Comment:
Consider re-exporting `glob` from `__init__`. This will make it available via `import hdfs` similar to the rest of the core (non-extension) functionality.

https://github.com/mtth/hdfs/blob/acfe0810cd5d410783a66c19643a71db78e71b14/hdfs/__init__.py#L6-L8